apps/backend/src/eventHandlers/logging.ts

```diff
@@ -63,6 +69,7 @@ export default function createLoggingHandler() {
           break;
         case Event.EMAIL_INVITE:
         case Event.EMAIL_INVITES:
+        case Event.INVITE_ACCEPTED:
```

Comment:
`INVITE_ACCEPTED` and `PROPOSAL_INVITE_ACCEPTED` can be confusing in my opinion. Suggestion: a comment here explaining the difference might help.